assets/ds-moon-preset.js

```diff
@@ -123,6 +123,11 @@ module.exports = {
       },
       animation: {
         loader: "loader 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite",
+        rightslide: "rightslide 0.2s ease-in-out",
```

Comment:
Is this present defined in Moon React also? Can it be imported from there?

Reply:
yeah, it's defined there the same way. we do not copy the ds-moon-preset.js for now, bc of few errors got when tried to

Reply:
Also coupling together two repositories does not seem like a good idea, especially if they are derivative siblings of a source (the MDS Figma).